### DISCUSSION TITLE
Math Blocks in v7

### DISCUSSION THREAD

**Discussion by sswatson** [Milkdown/milkdown#940] [Q&A] [answered]
I'm not sure how to create a math block now that the custom node from v6 is no longer available. If I do `$$ ` then it does go into math block mode, but I don't see how to input the LaTeX code at that point. It seems to render a zero-height element and de-focus the editor.

**Comment by Saul-Mirone** [comment, marked as answer]
You need to implement a view to enable the inline editing feature. Like what I did in:
https://github.com/Milkdown/website/blob/main/src/component/Playground/EditorComponent/MathBlock.tsx

**Reply by sswatson** [reply]
Thanks; I was wondering if that was the case.

I'm using Milkdown in Svelte, and I understand that only React and Vue are supported for creating views using the framework. But is it possible to do this in VanillaJS (or otherwise get this functionality in a non-React, non-Vue context)?
